filters/auth/webhook_test.go

```diff
@@ -73,7 +80,12 @@ func TestWebhook(t *testing.T) {
 				tok = tok[len(authHeaderPrefix):]
 				switch tok {
 				case testToken:
-					w.WriteHeader(200)
+					if ti.invalidScope {
+						w.WriteHeader(http.StatusForbidden)
+						fmt.Fprintln(w, "Forbidden - Got token: "+tok)
+						return
+					}
+					w.WriteHeader(http.StatusOK)
```

Comment:
Lets add a `testInvalidScopeToken` and a case for it instead of `invalidScope` test case field.
This way we could add other types of failure tokens that result in different responses without extending test case struct.

Also please use `t.Lof` for logging purposes (there is another `fmt.Fprintln` below - it could be fixed as well)

Reply:
The `testWebHookInvalidScopeToken` could be scoped to the `TestWebhook` function for now.

Reply:
> Also please use t.Logf for logging purposes

Nevermind, just saw it writes response body :facepalm:

Reply:
I agree with using a different token without a struct change to check for invalid the status code should be enough.

Reply:
Thanks for the input! Updated the PR.